tabpy-server/tabpy_server/handlers/query_plane_handler.py

```diff
@@ -116,6 +102,22 @@ def _handle_result(self, po_name, data, qry, uid):
 
             return (None, None)
 
+    def _sanitize_request_data(self, data):
+        if not isinstance(data, dict):
+            msg = 'Expect input data to be a dictionary'
+            self.logger.log(logging.CRITICAL, msg)
+            raise RuntimeError(msg)
+
+        if "method" in data:
+            return {"data": data.get("data"), "method": data.get("method")}
+        elif "data" in data:
+            return data.get("data")
+        else:
+            msg = ('Expect input data is a dictionary with at least a '
```

Comment:
Nit:
 msg = 'Input data must be a dictionary with a key called "data"'

Reply:
fixed